doc/coding-style.md

```diff
@@ -1,40 +1,37 @@
+1. We avoid `this.` unless absolutely necessary. 
+1. We always specify the visibility, even if it's the default (i.e.
    `private string _foo` not `string _foo`). Visibility should be the first modifier (i.e. 
    `public abstract` not `abstract public`).
-6. Namespace imports should be specified at the top of the file, *outside* of
+1. Namespace imports should be specified at the top of the file, *outside* of
    `namespace` declarations and should be sorted alphabetically.
-7. Avoid more than one empty line at any time. For example, do not have two
+1. Avoid more than one empty line at any time. For example, do not have two
    blank lines between members of a type.
-8. Avoid spurious free spaces.
+1. Avoid spurious free spaces.
    For example avoid `if (someVar == 0)...`, where the dots mark the spurious free spaces.
    Consider enabling "View White Space (Ctrl+E, S)" if using Visual Studio, to aid detection.
-9. If a file happens to differ in style from these guidelines (e.g. private members are named `m_member`
+1. If a file happens to differ in style from these guidelines (e.g. private members are named `m_member`
    rather than `_member`), the existing style in that file takes precedence.
-10. We only use `var` when it's obvious what the variable type is (i.e. `var stream = new FileStream(...)` not `var stream = OpenStandardInput()`).
-11. We use language keywords instead of BCL types (i.e. `int, string, float` instead of `Int32, String, Single`, etc) for both type references as well as method calls (i.e. `int.Parse` instead of `Int32.Parse`). See issue [391](https://github.com/dotnet/corefx/issues/391) for examples.
-12. We use PascalCasing to name all our constant local variables and fields. The only exception is for interop code where the constant value should exactly match the name and value of the code you are calling via interop.
-13. We use ```nameof(...)``` instead of ```"..."``` whenever possible and relevant.
-14. Fields should be specified at the top within type declarations.
-15. When including non-ASCII characters in the source code use Unicode escape sequences (\uXXXX) instead of literal characters. Literal non-ASCII characters occasionally get garbled by a tool or editor.
-
-We have provided a Visual Studio 2013 vssettings file (`corefx.vssettings`) at the root of the corefx repository, enabling C# auto-formatting conforming to the above guidelines. Note that rules 7 and 8 are not covered by the vssettings, since these are not rules currently supported by VS formatting.
-
-We also use the [.NET Codeformatter Tool](https://github.com/dotnet/codeformatter) to ensure the code base maintains a consistent style over time, the tool automatically fixes the code base to conform to the guidelines outlined above.
```

Comment:
If removing .NET Codeformatter, is there an alternate VS linter that we can use for validating coding style of PRs.

Reply:
VS does a pretty good job of that these days, I think. The question is what settings you use. One option is CodeMaid, an add-in that sets your settings, and has a file you can commit to your repo so that anyone else with CodeMaid picks them up automatically when loading solutions from your repo.